### ISSUE TITLE
App crash Fatal Exception: kotlin.KotlinNullPointerException        at getShouldDrawCount (PlutoIndicator.kt:91)

### ISSUE THREAD

**Issue by amjadmasri** [OpenSooq/Pluto#11]
new version of our app has launched yesterday using your library 
and a lot of crashes happened really fast 
here is the crash 
```
Fatal Exception: kotlin.KotlinNullPointerException
       at com.opensooq.pluto.PlutoIndicator.getShouldDrawCount + 91(PlutoIndicator.kt:91)
       at com.opensooq.pluto.PlutoIndicator.redraw + 403(PlutoIndicator.kt:403)
       at com.opensooq.pluto.PlutoIndicator$initObserver$1.onChanged + 174(PlutoIndicator.kt:174)
       at androidx.recyclerview.widget.RecyclerView$AdapterDataObservable.notifyChanged + 11997(RecyclerView.java:11997)
       at androidx.recyclerview.widget.RecyclerView$Adapter.notifyDataSetChanged + 7070(RecyclerView.java:7070)
       at com.opensooq.pluto.base.PlutoAdapter.clear + 47(PlutoAdapter.kt:47)
```

we have set  app:indicator_visibility="false"  //this line didn't make the indicators disappear  on its own so 
also onCreate we have set plutoView.setIndicatorVisibility(false);

any help would be appreciated thanks

**Comment by amjadmasri** [comment]
@Ramzi-Alqrainy @tamtom 
turns out that this is happening because of configuration changes and the activity being recreated 
what do you suggest for handling this ? 
I thought that recreating the activity would create a new instance 
but apparently the old instance remains with null recyclerView ? 
can  you help me please ?

**Comment by tamtom** [comment]
@amjadmasri  fixed in 1.6
